README.md

```diff
@@ -7,36 +7,28 @@
 [![Release](https://img.shields.io/github/release/zalando/logbook.svg)](https://github.com/zalando/logbook/releases)
 [![Maven Central](https://img.shields.io/maven-central/v/org.zalando/logbook-parent.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/logbook-parent)
 
-*Logbook* is an extensible library to enable complete request and response logging for different client- and server-side 
-technologies. It satisfies a special need by allowing web application developers to log any HTTP traffic that an application
-receives or sends in a way that makes it easy to persist and analyze it later. This can be used for traditional log analysis,
-for audit requirements or for investigating individual historic traffic issues. *Logbook* is ready to use out of the box for
-most common setups, but even for uncommon applications and technologies it should be very easy to implement the
-necessary interfaces to connect some library/framework/etc. to Logbook.
-
-## Features
-
-- logging of HTTP requests and responses **including the body**
-- customizable conditions which requests to log
-- optional obfuscation of sensitive data
-- customizable logging format
-- customizable logging destination
-- support for Servlet containers
-- partial logging (no body) for unauthorized requests
-- support for Apache's HTTP Client
-- Spring Boot Auto Configuration
-- sensible defaults
-- simple API that makes it easy to support other frameworks
-
-## Dependencies
+**Logbook** is an extensible Java library to enable complete request and response logging for different client- and server-side technologies. It satisfies a special need by a) allowing web application developers to log any HTTP traffic that an application receives or sends b) in a way that makes it easy to persist and analyze it later. This can be useful for traditional log analysis, meeting audit requirements or investigating individual historic traffic issues. 
+
+Logbook is ready to use out of the box for most common setups. Even for uncommon applications and technologies, it should be simple to implement the necessary interfaces to connect a library/framework/etc. to it.
+
+### Features
+
+-  *Logging*: of HTTP requests and responses, including the body; partial logging (no body) for unauthorized requests
+-  *Customization*: of logging format, logging destination, and conditions that request to log

```

Comment:
Got it. So you could say: 

*Customization*: of logging format, logging destination, and any provided conditions that disable logging for certain requests (and responses)